.gitignore

```diff
@@ -17,4 +17,4 @@ docs/
 reports/*
 !reports/.empty
 
-node_modules/
+node_modules/
```

Comment:
can we get rid of any changes to this file?

Reply:
👍 done